static/js/services/live-list.js

```diff
@@ -330,8 +337,8 @@ angular.module('inboxServices').factory('LiveList',
     }
 
     function _update(listName, updatedItem) {
-      _remove(listName, updatedItem);
-      _insert(listName, updatedItem);
+      var removed = _remove(listName, updatedItem);
```

Comment:
Cool.
Could you just pass a `bool` `isChecked`, instead of passing the whole removedElement? (unless I'm missing something)